EventSource/EventSource.swift

```diff
@@ -44,6 +46,14 @@ public class EventSource: NSObject, NSURLSessionDataDelegate {
         self.receivedString = nil
         self.receivedDataBuffer = NSMutableData()
 
+
+		let port = self.url.port?.stringValue ?? ""
+		let relativePath = self.url.relativePath ?? ""
+		let host = self.url.host ?? ""
+
+		self.uniqueIdentifier = "\(self.url.scheme).\(host).\(port).\(relativePath)"
+		self.lastEventIDKey = "\(EventSource.DefaultsKey).\(self.uniqueIdentifier)"
```

Comment:
but i would have to "calculate the key" every time a new event arrives... this way I just create the key and use it when I need it.